### ISSUE TITLE
Screen can't be changed

### ISSUE THREAD

**Issue by ml-** [ValveSoftware/Proton#521]
[Steam - System Information (Gist)](https://gist.github.com/ml-/5927742b82e2371048e31b463550818f)

**My screen setup:**

1. 3200x1800 (Laptop)
2. 1920x1080 (external Monitor) [right of 1]

**Problem:**

Game (e.g. GTA:SA) always starts on primary screen (1).
I couldn't find a way to change the screen used by the game.

Which component decides what screen to use for the game? Proton/Wine or the game itself?
Are there parameters I can pass to Proton to force a screen?

**Comment by insunaa** [comment]
Usually your window manager or compositor decides that...

**Comment by ml-** [comment]
@d3rrial Although my window manager is not very sophiscated, I tried setting initial position and ignoring initial settings. Nothing changes as if the window doesn't care about the settings.

Do you have a similar setup and can confirm you can configure the screen?

**Comment by insunaa** [comment]
@ml- Sorry, no, I use Gnome right now, stuff just works for me.

**Comment by ml-** [comment]
@d3rrial So what is it. You don't have a external monitor to test it. Or do you have one and it works?

**Comment by insunaa** [comment]
I have 2 monitors, games always start on the primary monitor, but I can just grab it with the mouse and drag it to the other screen and then it's rendered there

**Comment by pchome** [comment]
> Are there parameters I can pass to Proton to force a screen?

`$ DISPLAY=:0.1 someapp` ?
_format: "hostname:D.S", commonly used value `:0`_

**Comment by ml-** [comment]
@pchome Right now it can't find a display under `:0.1`. But thanks for that information, now I know where to start looking.

**Comment by Qix-** [comment]
Check [this SO answer](https://askubuntu.com/a/432257) on the XOrg `DISPLAY` environment variable. Unless you're not using XOrg (which is uncommon for Linux), this is how you would adjust the monitor selection. It's not a given than `:0.1` will work - it depends on how XOrg is configured.

**Comment by ml-** [comment]
@Qix- Yes, that's the problem. Also, my Window manager doesn't seem to handle such a thing (JWM).

For completeness sake, also valuable information here: https://wiki.archlinux.org/index.php/Multihead#Full_screen_games
